### PR TITLE
Converts html pages to handle bars

### PR DESCRIPTION
**Pull request by julianfortune**
- Converts everything to handlebars.
- Makes a bunch of little tweaks (oops)
- Tweaks fonts
- Had an issue with paths with `/`s and the css files, so I added `get`s for both css files in the `server.js`

### CODE REVIEW

**Comment by julianfortune** [comment]
Also increases the number of sample quiz cards to like a dozen

File: exampleQuiz.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        "answer": "Atlanta"
+    }, {
+        "prompt": "Hawaii",
+        "answer": "Honolulu"
+    }, {
+        "prompt": "Idaho",
+        "answer": "Boise"
+    }, {
+        "prompt": "Illinois",
+        "answer": "Springfield"
```
Comment (julianfortune) [inline exampleQuiz.json:48]: Ripped from https://gist.github.com/jpriebe/d62a45e29f24e843c974

**Review by thuyvyng — CHANGES_REQUESTED** [review]
card count is inaccurate

**Review by thuyvyng — APPROVED** [review]

**Review by tobsbird — APPROVED** [review]
I like

**Pull request merged**